### ISSUE TITLE
TSLint and ESLint both running on JS file (in non-TS project)

### ISSUE THREAD

**Issue by smlombardi** [Microsoft/vscode#16404]
- VSCode Version: Code - Insiders 1.8.0-insider (daa71ca002722c2778dc30bba151d7316e3c5e54, 2016-12-02T07:05:15.686Z)
- OS Version: Darwin x64 16.1.0
- Extensions:

|Extension|Author|Version|
|---|---|---|
|material-icon-theme|PKief|1.1.4|
|code-settings-sync|Shan|2.3.8|
|sort-lines|Tyriar|1.1.0|
|html-css-class-completion|Zignd|1.0.3|
|Bookmarks|alefragnani|0.10.0|
|project-manager|alefragnani|0.12.2|
|one-monokai|azemoh|0.2.3|
|vscode-eslint|dbaeumer|1.1.0|
|githistory|donjayamanne|0.1.2|
|tslint|eg2|0.6.7|
|vscode-great-icons|emmanuelbeziat|1.1.26|
|Angular2|johnpapa|1.0.2|
|theme-karyfoundation-themes|karyfoundation|10.0.0|
|HTMLHint|mkaufman|0.0.3|
|vscode-stylefmt|mrmlnc|2.2.0|
|Theme-1337|ms-vscode|0.1.2|
|angular2-inline|natewallace|0.0.16|
|nonbreakingspace|smlombardi|0.0.1|
|vscode-icons|robertohuertasm|4.0.4|
|stylelint|shinnn|0.20.3|
|theme-tesla|smlombardi|5.3.4|
|bootstrap-3-snippets|wcwhitehead|0.0.8|
|change-case|wmaurer|1.0.0|

---

Steps to Reproduce:

1. I have a project I've been working on for weeks in Insiders. This is a html/js/scss project -- no typescript.

Today I noticed some new errors reported in the right scrollbar and the status bar counter. For rules I had not set in ESLint config.

Then I noticed the status bar said both ESLint and TSLint were active:
![screen shot 2016-12-02 at 2 45 10 pm](https://cloud.githubusercontent.com/assets/835148/20848023/598f795e-b89e-11e6-9595-259c62252480.png)

In previous builds it only displayed ESLint.

**Comment by mjbvz** [comment]
@dbaeumer, I know you did some work on eslint recently to change the types of files it picks up. It this related?

**Comment by dbaeumer** [comment]
No, this is actually tslint related. @smlombardi complains about that now TSLint and ESLint shows up where in previous releases only ESLint was correctly shown.

@egamma any changes that make TSLint activate on JS files ?

**Comment by egamma** [comment]
@dbaeumer tslint 4.0 supports to lint .js files https://palantir.github.io/tslint/2016/11/17/new-for-4.0.html so it is now also activated for .js files.

**Comment by dbaeumer** [comment]
@egamma cool. But I think we need an option to disable this. If someone wants to use ESLint for JS and TSLint for TS we should only activate TSLint when TS files are active. 

This might require TSLint adopting the latest LSP node libraries to have an easier way to control this.

I opened https://github.com/Microsoft/vscode-tslint/issues/149.

Closing as upstream since there is nothing VS Code can do to fix this.
